test/python/filters_IECPU.py

```diff
@@ -0,0 +1,147 @@
+# ==============================================================================
```

Comment:
Can we turn this into a .txt manifest file just for consistency's sake?

Reply:
With .py extension it also gets automatically copied to artifacts folder during install. Also, while editing/adding content the vscode-IDE will highlight any syntax errors quickly.

Reply:
Manifest files don't need syntax highlighting.